extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -366,6 +368,7 @@ export class SKURecommendationPage extends MigrationWizardPage {
 			this._resourceDropdown.loading = true;
 			try {
 				this._managedInstanceSubscriptionDropdown.values = await this.migrationStateModel.getSubscriptionsDropdownValues();
+				this._managedInstanceSubscriptionDropdown.value = this._managedInstanceSubscriptionDropdown.values[0];
```

Comment:
any concerns values will be an empty list and get out of bounds index here?

Reply:
This method always return a value even if subscriptions are not found.